### ISSUE TITLE
Provider produced inconsistent result after apply

### ISSUE THREAD

**Issue by awolfson2** [PropelAuth/terraform-provider-propelauth#18]
### Description
Terraform Apply results in an error that states an issue with the provider.  I want to use the "SplitScreen" layout with content_type "None". I've tried this with both manually defining the defaults and leaving the `split_login_page_parameters` stanza empty, but both render the same error.

### Setup in Terraform
```hcl
resource "propelauth_theme" "login" {
  body_font = "Inter"
  login_page_theme = {
    layout = "SplitScreen"
    split_login_page_parameters = {
    }

    background_type = "Image"
    image_background_parameters = {
      default_background_color = "#f7f7f7"
    }
    frame_background_color  = "#ffffff"
    frame_text_color        = "#0f0f0f"
    primary_color           = "#fb0750"
    primary_text_color      = "#f7f7f7"
    error_color             = "#cf222e"
    error_button_text_color = "#ffffff"
    border_color            = "#e4e4e4"
  }
  management_pages_theme = {
    main_background_color    = "#f7f7f7"
    main_text_color          = "#363636"
    navbar_background_color  = "#ffffff"
    navbar_text_color        = "#0f0f0f"
    action_button_color      = "#fb0750"
    action_button_text_color = "#f7f7f7"
    border_color             = "#e4e4e4"
  }
}
```
### Terraform plan expectation
```hcl
  # module.propelauth.propelauth_theme.login will be created
  + resource "propelauth_theme" "login" {
      + body_font              = "Inter"
      + display_project_name   = true
      + header_font            = "Inter"
      + login_page_theme       = {
          + background_type             = "Image"
          + border_color                = "#e4e4e4"
          + error_button_text_color     = "#ffffff"
          + error_color                 = "#cf222e"
          + frame_background_color      = "#ffffff"
          + frame_text_color            = "#0f0f0f"
          + image_background_parameters = {
              + background_text_color    = "#363636"
              + default_background_color = "#f7f7f7"
            }
          + layout                      = "SplitScreen"
          + primary_color               = "#fb0750"
          + primary_text_color          = "#f7f7f7"
          + split_login_page_parameters = {
              + content_type                    = "None"
              + direction                       = "Left"
              + secondary_background_text_color = "#363636"
            }
        }
      + management_pages_theme = {
          + action_button_color      = "#fb0750"
          + action_button_text_color = "#f7f7f7"
          + border_color             = "#e4e4e4"
          + display_navbar           = true
          + main_background_color    = "#f7f7f7"
          + main_text_color          = "#363636"
          + navbar_background_color  = "#ffffff"
          + navbar_text_color        = "#0f0f0f"
        }
    }
```

### Actual Result
Error:
```bash
│ Error: Provider produced inconsistent result after apply
│ 
│ When applying changes to module.propelauth.propelauth_theme.login, provider
│ "module.propelauth.provider[\"registry.terraform.io/propelauth/propelauth\"]" produced an unexpected new value:
│ .login_page_theme.split_login_page_parameters.header: was null, but now cty.StringVal("").
│ 
│ This is a bug in the provider, which should be reported in the provider's own issue tracker.
╵
╷
│ Error: Provider produced inconsistent result after apply
│ 
│ When applying changes to module.propelauth.propelauth_theme.login, provider
│ "module.propelauth.provider[\"registry.terraform.io/propelauth/propelauth\"]" produced an unexpected new value:
│ .login_page_theme.split_login_page_parameters.subheader: was null, but now cty.StringVal("").
│ 
│ This is a bug in the provider, which should be reported in the provider's own issue tracker.
```

**Comment by mrmauer** [comment]
Hey @awolfson2 , thanks for flagging this. We'll take a closer look and circle back here.

**Comment by mrmauer** [comment]
@awolfson2 The fix for this shipped in 0.1.3. Let us know if you see any further issue!

**Comment by awolfson2** [comment]
i'll give it a go! thanks @mrmauer
